app/controllers/admin/instances/notes_controller.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class Admin::Instances::NotesController < Admin::BaseController
+  before_action :set_instance, only: [:create]
+  before_action :set_instance_note, only: [:destroy]
+
+  def create
+    authorize :instance_note, :create?
+
+    @instance_note = current_account.instance_notes.new(content: resource_params[:content], domain: @instance.domain)
+
+    if @instance_note.save
+      redirect_to admin_instance_path(@instance.domain, anchor: helpers.dom_id(@instance_note)), notice: I18n.t('admin.instances.notes.created_msg')
```

Comment:
I think the usage of the anchors here is a nice touch, though a little non-standard with how we've done this elsewhere.